### DISCUSSION TITLE
Formatting a solved problem

### DISCUSSION THREAD

**Discussion by AaphexTwin** [codewars/codewars.com#3187] [General]
Hello! Tell me how to correctly format the solution to the problem. I did everything right, but I don’t know what to add to make it work. maybe you need to use 'return' somehow.
`import math
def strong_num(n):
    k = []
    a = 0
    b = n
    while n > 0:
        k.append(n % 10)
        n = n // 10
        a += (math.factorial(k[0]))
        k.pop(0)
    if a == b:
        print("STRONG!!!!")
    else:
        print("Not Strong !!")
strong_num(int(input()))
https://www.codewars.com/kata/5a4d303f880385399b000001/train/python - exercise
`

**Comment by AaphexTwin** [comment]
`
import math


def  strong_num(n) :
    k = []
    a = 0
    b = n
    while n > 0:
        k.append(n % 10)
        n = n // 10
        a += (math.factorial(k[0]))
        k.pop(0)

    if a == b:
        print("STRONG!!!!")
    else:
        print("Not Strong !!")


strong_num(int(input()))
`

**Reply by hobovsky** [reply]
[This article](https://docs.codewars.com/training/training-example) should answer your question.

**Reply by SJM724** [reply]
You don't need to call the function, just replace the print() statements with return statements.
